app/models/user_column.rb

```diff
@@ -0,0 +1,13 @@
+class UserColumn < ApplicationRecord
+  include Authorizable
+  include Exportable
```

Comment:
Is this needed? do we have any use case for exporting user columns?

Reply:
that came because of  https://github.com/theforeman/foreman/pull/5262#discussion_r169331051